modules/fedimint-ln-common-ng/src/db.rs

```diff
@@ -0,0 +1,127 @@
+use bitcoin_hashes::sha256;
```

Comment:
This file should be moved to `fedimint-ln-server-ng`, we shouldn't share db structures between client and server (recently fixed this in the legacy implementation as well)